JS/app.js

```diff
@@ -0,0 +1,123 @@
+// Book class: Represent book
+class Book {
+  constructor(title, author, pages, type) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.type = type;
+  }
+}
+// UI class: Handle UI task
+class UI {
+  static displayBooks() {
+    const StoredBooks = [
+      {
+        title: 'Harry Potter',
+        author: 'John Doe',
+        pages: '234',
+        type: 'Read',
+      },
+      {
+        title: 'Sherlock Hormes',
+        author: 'Dr John Doe',
+        pages: '587',
+        type: 'Unread',
+      },
+      {
+        title: 'Tale of Two Cities',
+        author: 'Dr Smith Doe',
+        pages: '359',
+        type: 'Unread',
+      },
+    ];
+    const books = StoredBooks;
+    books.forEach(book => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.getElementById('tableBody');
+    const row = `<tr>
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <td>${book.pages}</td>
+    <td>${book.type}</td>
+    <td><a href='#' class='btn btn-danger btn-sm delete'>X</a></td> 
+    </tr>`;
+    list.innerHTML += row;
+  }
```

Comment:
Regarding of the change required below, we are missing a button that will toggle the read status of the book, please make sure you implement that as well.